### PR TITLE
Python2 updates

### PR DESCRIPTION
**Pull request by peasant98**
Now things work in python2.
Addresses Issue #64

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,7 @@
...
 # install the necessary packages
-RUN apt-get update && apt-get install -y python3-pip python3-yaml \
-    && sudo pip3 install rospkg catkin_pkg
-
-COPY . .
+RUN apt update && apt -y install python-pip
+RUN mkdir robotic_skin
+COPY . robotic_skin/
+RUN ls robotic_skin
+RUN cd robotic_skin && pip install .
```
Comment (krishnachaitanya7) [inline Dockerfile]: This obviously works no doubt. But it's not the best docker practice. Always make as few layers as possible. Each command line you add, adds one extra layer.

File: requirements.txt [hunk 2]
```diff
@@ -2,12 +2,17 @@ Cython
...
+smbus2
+VL53L1X
+pyquaternion
+flake8
+Cython
+future
+matplotlib
+numpy
+mcp3208
 nlopt
```
Comment (krishnachaitanya7) [inline requirements.txt]: You kind of repeated requirements double times for some. Can you please remove duplicates?

**Review by krishnachaitanya7 — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (peasant98) [reply Dockerfile]: fixed.

File: requirements.txt (hunk 2 above)
Comment (peasant98) [reply requirements.txt]: yep

**Review by krishnachaitanya7 — APPROVED** [review]

**Pull request merged**